<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoveMeter</class>
 <widget class="QDialog" name="LoveMeter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>614</width>
    <height>250</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLCDNumber" name="lcdNumber">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>100</y>
     <width>131</width>
     <height>71</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>100</y>
     <width>361</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QProgressBar {
     border: 3px solid grey;
     border-radius: 5px;

 }

 QProgressBar::chunk {
     background-color: #ff449f;
     width: 20px;
 }</string>
   </property>
   <property name="value">
    <number>17</number>
   </property>
   <property name="textVisible">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_compat">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>30</y>
     <width>491</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ageo</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
color:darkblue</string>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="clearButtonEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>190</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ageo</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white</string>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>621</width>
     <height>251</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/new/prefix1/lovemeter.png</pixmap>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>progressBar</zorder>
  <zorder>lineEdit_compat</zorder>
  <zorder>pushButton</zorder>
  <zorder>lcdNumber</zorder>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>LoveMeter</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>323</x>
     <y>213</y>
    </hint>
    <hint type="destinationlabel">
     <x>458</x>
     <y>211</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
